curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -485,7 +485,6 @@ private String protectedPathInForeground(String adjustedPath, byte[] data) throw
         }
         catch ( Exception e)
         {
-            ThreadUtils.checkInterrupted(e);
```

Comment:
Won't the removal of this call mean that the thread will not be marked as interrupted unless the caller catches the exception and resets the value themselves? This code is called from many recipes and can also be called explicitly by users Curator code. I would think that there's great potential to break code here, unless I'm missing something.